src/main/java/games/strategy/engine/ClientFileSystemHelper.java

```diff
@@ -21,9 +21,7 @@
 import games.strategy.triplea.settings.GameSetting;
 
 /**
- * Pure utility class, final and private constructor to enforce this
- * WARNING: do not call ClientContext in this class. ClientContext call this class in turn
- * during construction, depending upon ordering this can cause an infinite call loop.
```

Comment:
This circular dependency appears to no longer be true.  I suspect whatever was causing it was removed when `ClientContext` was refactored several months ago.